tests/src/test/scala/com/twitter/intellij/pants/OpenProjectTest.scala

```diff
@@ -66,10 +58,9 @@ class OpenProjectTestBsp extends CommonOpenProjectTests {
   // TODO add to common OpenProjectTest when it works with pants
   @Test def hasGitRepositoryRootDetected(): Unit = {
     val actualVcsRoots = intelliJ.probe.vcsRoots()
-    val expectedRoot = VcsRoot("Git", intelliJ.workspace)
+    val expectedRoot = VcsRoot("Git", intelliJ.workspace.toRealPath())
```

Comment:
If intelliJ.workspace is not a cannonical path, we might want to put this logic in probe itself and remove it here. It can be done in a way that we merge this as is, but open PR to probe with this change and remove it from here later.